.github/workflows/autoclose-issues.yml

```
@@ -0,0 +1,54 @@
+name: Autocloser
+on:
+  issues:
+    types: [opened, edited]
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - id: match-title
+        name: Match title
+        # if the title contains the word Java (case insensitive)
+        # we make sure that this word is the end of the string or is followed by a space character (ex. we do not want to match javascript)
+        # we make sure that this word is the beginning of the string or is preceded by a space character (ex. we do not want to match foojava)
+        run: |
+          if [[ ${{ github.event.issue.title }} =~ (^|[[:space:]])([j|J][a|A][v|V][a|A])([[:space:]]|$) ]]; then
```

```suggestion
          if [[ ${{ github.event.issue.title }} =~ (^|[[:space:]])([jJ][aA][vV][aA])([[:space:]]|$) ]]; then
```

This regex will match `||||`. The `|` will be treated as a character when put to [character classes](https://www.rexegg.com/regex-quickstart.html#classes).